### PR TITLE
cargo-xbuild: 0.5.12 -> 0.5.13

### PR DESCRIPTION
**Pull request by xrelkd**
###### Motivation for this change
Update `cargo-xbuild` to the latest release.
- https://github.com/rust-osdev/cargo-xbuild/releases/tag/v0.5.13
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/rust/cargo-xbuild/default.nix [hunk 1]
```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, rustPlatform }:
+{ lib, fetchFromGitHub, rustPlatform }:
```
Comment (FRidh) [inline pkgs/development/tools/rust/cargo-xbuild/default.nix]: Please don't make these unnecessary changes.

Comment (xrelkd) [reply pkgs/development/tools/rust/cargo-xbuild/default.nix]: Updated as your suggestion, thank you!
